ramp-database/ramp_database/tests/test_cli.py

```diff
@@ -198,6 +198,40 @@ def test_add_event(make_toy_db):
     assert result.exit_code == 0, result.output
 
 
+def test_delete_problem(make_toy_db):
+    runner = CliRunner()
+    ramp_config = read_config(ramp_config_template())
+    runner.invoke(
+        main,
+        [
+            "add-problem",
+            "--config",
+            database_config_template(),
+            "--problem",
+            "iris",
+            "--kit-dir",
+            ramp_config["ramp_kit_dir"],
+            "--data-dir",
+            ramp_config["ramp_data_dir"],
+            "--force",
+            True,
+        ],
+        catch_exceptions=False,
+    )
+    result = runner.invoke(
+        main,
+        [
+            "delete-problem",
+            "--config",
+            database_config_template(),
+            "--problem",
+            "iris",
+        ],
+        catch_exceptions=False,
+    )
```

Comment:
any chance you can assess in the database that the problem is available after you add it and then gone once you remove it? see how it's done elsewhere.